spec/static/openapitransactional.yaml

```diff
@@ -2379,8 +2300,8 @@ paths:
         in the link.
       operationId: getSearchSTAC
       parameters:
-      - $ref: "#/components/parameters/assetQuery"
-      - $ref: "#/components/parameters/query"
+      - #- $ref: "#/components/parameters/assetQuery"
+        $ref: "#/components/parameters/query"
```

Comment:
why no "-" here anymore?

Reply:
the - comes from above ... as autogenerated